parachain/pallets/ethereum-beacon-client/src/mock.rs

```diff
@@ -10,6 +10,10 @@ use snowbridge_core::inbound::{Log, Proof};
 use sp_core::H256;
 use sp_runtime::traits::{BlakeTwo256, IdentityLookup};
 use std::{fs::File, path::PathBuf};
+#[cfg(feature = "beacon-spec-minimal")]
```

Comment:
Nit: add a new line before this block

Reply:
https://github.com/Snowfork/snowbridge/pull/1102/commits/16f73df03bafa277c5039a26fba91096cc2fa110 :)